src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,34 @@
+import { Person } from '../types';
+import PersonLink from './PersonLink';
+
+type Props = {
+  people: Person[] | null;
+};
+
+const PeopleTable: React.FC<Props> = ({ people }) => {
+  const showPeople = people?.map(person => {
+    return <PersonLink person={person} people={people} key={person.slug} />;
+  });
```

Comment:
The 'key' prop provided to the list items should be a unique identifier. Ensure that 'person.slug' is unique across all people; otherwise, React will not be able to manage the list efficiently, which can lead to rendering issues.